kalite/coachreports/static/js/coachreports/coach_reports/router.js

```diff
@@ -0,0 +1,39 @@
+CoachReportRouter = Backbone.Router.extend({
+    initialize: function(options) {
+        _.bindAll(this);
+        this.state_model = new Backbone.Model();
+        this.listenTo(this.state_model, "change", this.set_url);
+    },
+
+    routes: {
+        "(:facility/)(:group/)(:report/)":    "report"
```

Comment:
Can you add some examples of valid urls? Including one where the `:facility` parameter isn't included?